src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/HttpBasicFactory.php

```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\DependencyInjection\DefinitionDecorator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\Security\Http\EntryPoint\BasicAuthenticationEntryPoint;
```

Comment:
We don't need to update this one, BasicAuthenticationListener supports [any `AuthenticationEntryPointInterface`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Http/Firewall/BasicAuthenticationListener.php#L36). Only digest is tied to a specific implementation.

Reply:
I tried to replace my digest authentication by basic and I had the same issue. That's why I also modified this class.

Reply:
Strange, I'll try it. Thanks